modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsValidationsDisplayContext.java

```diff
@@ -96,7 +96,7 @@ public List<Map<String, String>> getObjectValidationRuleEngines() {
 					objectValidationRuleEngine.getLabel(
 						objectRequestHelper.getLocale())
 				).put(
-					"name", objectValidationRuleEngine.getKey()
+					"value", objectValidationRuleEngine.getKey()
```

Comment:
@danvb112 Following the existing pattern, this property should be called "key".

Reply:
@naraandrade, same comment as above